metadata-ingestion/docs/dev_guides/classification.md

```diff
@@ -6,43 +6,60 @@ The classification feature enables sources to be configured to automatically pre
+| info_types_config.`key`.datatype.type                  |                                                       | Array of string                                | List of data types for the info type                                                                                                                      | ['.*']                                                                                                                                                                |
+| info_types_config.`key`.values                         |                                                       | ValuesFactorConfig (see below for fields)      |                                                                                                                                                           |                                                                                                                                                                       |
+| info_types_config.`key`.values.prediction_type         | ❓ (required if info_types_config.`key`.values is set) | string                                         |                                                                                                                                                           | None                                                                                                                                                                  |
+| info_types_config.`key`.values.regex                   |                                                       | Array of string                                | List of regex patterns the column value follows for the info type                                                                                         | None                                                                                                                                                                  |
+| info_types_config.`key`.values.library                 |                                                       | Array of string                                | Library used for prediction                                                                                                                               | None                                                                                                                                                                  |
+| minimum_values_threshold                               |                                                       | number                                         | Minimum number of non-null column values required to process `values` prediction factor.                                                                  | 50                                                                                                                                                                    |
```

Comment:
Added new config param minimum_values_threshold and list of supported info types below.